### PR TITLE
Update instructions in Gemini notebook

### PR DESCRIPTION
**Pull request by BeibinLi**
## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: notebook/agentchat_gemini.ipynb [hunk 1]
```diff
@@ -7,15 +7,45 @@
...
+    "pip install \"google-generativeai\" \"pydash\" \"pillow\" \"pydantic==1.10.13\"\n",
+    "```\n",
+    "\n",
+    "## Features\n",
     "You don't need to handle OpenAI or Google's GenAI packages. AutoGen already handled all of these for you.\n",
     "\n",
     "You can just create different agents with different backend LLM with assistant agent, and all models/agents are at your fingertip.\n",
     "\n",
     "\n",
+    "\n",
```
Comment (qingyun-wu) [inline notebook/agentchat_gemini.ipynb:26]: There are several empty code blocks in this notebook. Shall we remove them?

**Comment by qingyun-wu** [comment]
Also I see that this page is not included in the Example page: https://microsoft.github.io/autogen/docs/Examples. Do we want to include it?

**Comment by BeibinLi** [comment]
@qingyun-wu Thanks for the suggestions.

This PR is for the "gemini" branch, and I just included the notebook into the "Examples.md". It will be compiled for the website in the future when it is merged into the main branch.

**Comment by jackgerrits** [comment]
If you define the front matter itll be available rendered on the website too https://github.com/microsoft/autogen/blob/main/notebook/contributing.md#how-to-get-a-notebook-displayed-on-the-website

**Comment by BeibinLi** [comment]
> If you define the front matter itll be available rendered on the website too https://github.com/microsoft/autogen/blob/main/notebook/contributing.md#how-to-get-a-notebook-displayed-on-the-website

I don't think we want to include the notebook into the actual website yet. This branch is still experimental, and not ready for public use.

**Pull request merged**